packages/salesforcedx-utils-vscode/test/jest/helpers/paths.test.ts

```diff
@@ -70,4 +73,43 @@ describe('test project paths', () => {
       expect(relativeStateFolder).toEqual(FAKE_STATE_FOLDER);
     });
   });
+
+  describe('test SFDXGlobalConfigPath', () => {
+    it('should return the global config path', () => {
+      const mockOS = jest.requireActual('os');
+      mockOS.homedir = jest.fn().mockReturnValue(FAKE_HOME_DIR);
+      jest.mock('os', () => mockOS);
```

Comment:
jest.mock calls are hoisted to the top of the file by jest so that the execution will happen prior to anything else. See [here](jest.mock). Top avoid running into weird execution state issue I think we should always put the jest.mock calls at the top of the file directly after the imports (so on line 12 for this particular case).  Would you mind moving this mock call to the top of the file?  Does it make sense why I'm requesting it?

Reply:
yess. Thank you!!